docs/install.md

```diff
@@ -150,6 +139,123 @@ on the Kubernetes version.
 $ kubectl label node <your node> storage=pmem
 ```
 
+### Install PMEM-CSI driver
+
+PMEM-CSI driver can be deployed to a Kubernetes cluster either using the
+PMEM-CSI operator or by using reference yaml files provided in source code.
+
+#### Install using the operator
+
+[PMEM-CSI operator](./design.md#operator) facilitates deploying and managing
+the [PMEM-CSI driver](https://github.com/intel/pmem-csi) on a Kubernetes cluster.
+
+- **Setup deployment certificates**
+
+By default operator creates the needed private keys and certificates required
+for running the driver as described in [driver security](./design.md#security)
+section. Those certificates are generated by the operator using a self-signed CA.
+This can be overridden with custom keys and certificates by using appropriate fields
+in [deployment specification](#deploymentspec).These encoded certificates and 
+private keys are made available to driver pods via Kubernetes [secrets](https://kubernetes.io/docs/concepts/configuration/secret/)
+by the operator.
+
+**NOTE:** A production deployment that is not supposed to depend on the 
+operator's self-signed CA instead must provide the certificates generated
+from a trusted certificate authority.
+
+- **Install the PMEM-CSI operator**
+
+``` console
+$ kubectl create -f https://github.com/intel/pmem-csi/raw/devel/deploy/operator/pmem-csi-operator.yaml
+```
+
+- **Create a custom resource**
+
+Once the operator deployed in previous step is in `Running` state,
+it is ready to handle PMEM-CSI `Deployment` objects in `pmem-csi.intel.com`
+API group. Refer to [Deployment CRD API](#PMEM-CSI-Deployment-CRD) for 
+complete list of supported properties.
+
+Here is an example driver deployment by creating a custom resource:
```

Comment:
s/by creating a/created by a/

Reply:
Or "created with a" - perhaps better, it's not the CR which creates the deployment...

Reply:
Fixed.